.github/workflows/canary-e2e.yml

```diff
@@ -0,0 +1,12 @@
+name: Canary CI
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  canary-ci:
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/canary-ci run') }}
+    uses: dai-shi/waku/.github/workflows/e2e.yml@main
+    with:
+      react_version: 'canary'
```

Comment:
I would like to make `react_version` configurable with the comment.

```
/canary-ci run 19.1.0-canary-3b009b4c-20250102
```

At that point, it's not canary only, so we might want to change the command.

Reply:
Maybe just `canary-ci.yml`?

Reply:
updated the filename and added the support for the configurable with a comment run